h/models/organization.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_LENGTH = 4
+ORGANIZATION_NAME_MAX_LENGTH = 25
+ORGANIZATION_LOGO_MAX_LENGTH = 3000
+
+
+class Organization(Base, mixins.Timestamps):
+    __tablename__ = 'organization'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    logo = sa.Column(sa.UnicodeText())
+
+    def __init__(self, **kwargs):
+        super(Organization, self).__init__(**kwargs)
+
+    @property
+    def slug(self):
+        """A version of this organization's name suitable for use in a URL."""
+        return slugify.slugify(self.name)
+
+    @sa.orm.validates('name')
+    def validate_name(self, key, name):
+        if not ORGANIZATION_NAME_MIN_LENGTH <= len(name) <= ORGANIZATION_NAME_MAX_LENGTH:
+            raise ValueError('name must be between {min} and {max} characters '
+                             'long'.format(min=ORGANIZATION_NAME_MIN_LENGTH,
+                                           max=ORGANIZATION_NAME_MAX_LENGTH))
+        return name
+
+    @sa.orm.validates('logo')
+    def validate_logo(self, key, logo):
+        if not len(logo) <= ORGANIZATION_LOGO_MAX_LENGTH:
+            raise ValueError('logo must be less than {max} characters '
```

Comment:
I feel like this error message could potentially be confusing as it refers to logo as characters; OTOH I don't know what the plan is for UX for adding logos to an organization. Certainly we'll want to bake something clearer for user-facing errors, at the least! :)